src/Compilers/CSharp/Test/Semantic/Semantics/PropertyFieldKeywordTests.cs

```diff
@@ -82,6 +82,119 @@ private void VerifyTypeIL(CSharpCompilation compilation, string typeName, string
             CompileAndVerify(compilation).VerifyTypeIL(typeName, expected);
         }
 
+        [Fact]
+        public void TestInInterface()
+        {
+            var comp = CreateCompilation(@"
+public interface I
+{
+    public int P { get => field; }
```

Comment:
>public int P { get => field; } [](http://example.com/codeflow?start=4&length=30)

Consider also testing scenario with initializer.

Reply:
Is that fine with a prototype with now?

Reply:
> Is that fine with a prototype with now?

Can't we test this scenario right now? We don't really need the initializer to actually do any work because this is an error scenario anyway.